frontends/web/src/components/password.jsx

```diff
@@ -309,3 +304,8 @@ function MatchesPattern({ regex, value = '', text }) {
         <p style="color: var(--color-error);">{text}</p>
     );
 }
+
+function hasCaps(event) {
+    const char = String.fromCharCode(event.which);
+    return char.toUpperCase() === char && char.toLowerCase() !== char && !event.shiftKey;
```

Comment:
Can it actually happen that `char.toUpperCase() === char` and `char.toLowerCase() === char` at the same time?

Reply:
ehm.. sorry, never mind this.